packages/api/lambdas/sf-event-sqs-to-db-records/write-granules.js

```diff
@@ -306,18 +307,18 @@ const _writeGranuleFiles = async ({
       knex,
     });
   } catch (error) {
-    log.error('Failed writing some files to Postgres', error);
-
+    if (workflowError !== undefined) {
```

Comment:
I don't believe this works - error can be `{}` as a result of writeGranules's call to parseException.       

File this under 'I really want us to get the API code into TS ASAP'